### ISSUE TITLE
Include files in static build

### ISSUE THREAD

**Issue by darbio** [sanity-io/next-sanity#9]
I would like to include all my files and images (assets) in my build directory. 

Is there a way to download these from the CDN on build and include them in the status output?

It appears to have [been done with Gatsby](https://github.com/leanjs/gatsby-source-sanity-transform-images).

**Comment by stipsan** [comment]
> I think this would be a feature that most people would want to use?

Yes exactly :relaxed: When I said `use cases that most people that installs next-sanity will need` I didn't meant that your use case isn't generally useful to `next-sanity`. But that it's not uniquely useful, or something that should be resolved in `next-sanity` itself. For example we recently moved the live preview mode logic in `next-sanity` into its own [general-purpose library](https://github.com/sanity-io/preview-kit) as people writing Remix apps and other frameworks want to use it. If we were to build a library to solve your use-case what I'm saying is it should be a general library as I don't see a reason to why it should live in `next-sanity`.

The team that monitors this repo is prepared to answer questions that are very specific to Next.js features like On-demand incremental static regeneration, preview mode. While the question you asked is useful for anyone who builds something that targets users with challenging internet conditions. Could be Gatsby, could be astro. 

If you ask that question in the Community Slack you're guaranteed to have lots of other users who have asked this question before, and the team that monitors the Slack is well equipped and prepared to help you out. While I must admit that I don't have an answer for you, as I haven't faced this use case myself yet 😅
